network_diffusion/tests/test_experiment_logger.py

```diff
@@ -254,10 +254,8 @@ def test__convert_logs(self):
 
         logger.convert_logs(model_hyperparams)
         for phenomena, stat in logger._global_stats_converted.items():
-            self.assertTrue(
-                exp_logs_converted[phenomena].equals(stat),
-                f"Logs for phenomena {phenomena} incorrect. Expected value"
-                f" {exp_logs_converted[phenomena]}, got {stat}",
+            pd.testing.assert_frame_equal(
+                exp_logs_converted[phenomena], stat, check_dtype=False
             )
 
     def test__add_log(self):
```

Comment:
In addition to your changes, please bump the version of the library (I'd recommend `__version__ = "0.10.0"`). Moreover, please update a facade of the selectors: (`/network_diffusion/seeding/__init__.py`) to facilitate usage of the seed selection methods.